src/switchcraft/services/history_service.py

```
@@ -0,0 +1,60 @@
+            history = history[:100]
+
+        self._save(history)
+
+    def clear(self):
+        self._save([])
+
+    def _save(self, data):
+        try:
+            with open(self.history_file, 'w', encoding='utf-8') as f:
+                json.dump(data, f, indent=2)
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)